xml/System.CodeDom.Compiler/CodeDomProvider.xml

```diff
@@ -1493,7 +1493,7 @@
         <Parameter Name="supports" Type="System.CodeDom.Compiler.GeneratorSupport" Index="0" FrameworkAlternate="xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <param name="supports">To be added.</param>
+        <param name="generatorSupport">To be added.</param>
```

Comment:
@joelmartinez @dend I'm a bit confused on what's going on here. Why is the parameter name changing here and shouldn't .net-framework-4.7.2 also be on the FrameworkAlternate list on line 1492?

Reply:
Hmm ... yeah this is indeed strange. It shouldn't be _removing_ the `supports` parameter, it should only be adding `generatorSupport`. I've created a new GH issue, and will work to resolve:  
https://github.com/mono/api-doc-tools/issues/222